pkg/recipes/terraform/config/testdata/providers-envrecipedefaultconfig.tf.json

```diff
@@ -0,0 +1,30 @@
+{
+  "terraform": {
+    "backend": {
+      "kubernetes": {
+        "config_path": "/home/radius/.kube/config",
+        "namespace": "radius-system",
+        "secret_suffix": "test-secret-suffix"
+      }
+    }
+  },
+  "provider": {
+    "azurerm": [
+      null,
```

Comment:
Why do we have **null** here? Would/Why would a Radius User ever add **null** to the Config?

Reply:
this is covered in discussion in above comment and is now removed.